### PR TITLE
Updating jetty.test.helper repo

### PR DESCRIPTION
**Pull request by joakime**
fixing homepage and adding topics

### CODE REVIEW

**Comment by netomi** [comment]
the change wrt the homepage results now in conflicts due to the other PR that got merged.

File: otterdog/jetty.jsonnet [hunk 1]
```diff
@@ -298,7 +298,13 @@ orgs.newOrg('jetty') {
       dependabot_security_updates_enabled: true,
       description: "Eclipse Jetty® - Test Helper",
       has_wiki: false,
-      homepage: "https://eclipse.dev/jetty",
+      homepage: "https://jetty.org/",
```
Comment (netomi) [inline otterdog/jetty.jsonnet:301]: all the other repos have https://eclipse.dev/jetty as homepage. Not sure if this difference is by accident or intentional.

Comment (joakime) [reply otterdog/jetty.jsonnet:301]: Eh, not really an accident.
But it really doesn't matter, as the homepage is in transition, either URL is valid right now.

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
changes are live.

**Pull request merged**